presto-client/src/main/java/com/facebook/presto/client/FixJsonDataUtils.java

```diff
@@ -69,7 +69,10 @@ private FixJsonDataUtils() {}
         }
         requireNonNull(columns, "columns is null");
         List<TypeSignature> signatures = columns.stream()
-                .map(column -> parseTypeSignature(column.getType()))
+                .map(column -> {
+                    TypeSignature displayType = column.getTypeMetadata().getDisplayType();
```

Comment:
in this file, we use the type signature client-side to determine what kind of data we're dealing with.
Since the client doesn't know about the enum types, if it encounters a String value, it will default to base64 encoding as per:
```
                if (value instanceof String) {
                    return Base64.getDecoder().decode((String) value);
                }
```
For string-valued enums, this results in gibberish in the CLI output.
This is why I'm passing this extra info about the display type